src/inet_cidr.erl

```diff
@@ -9,64 +9,78 @@
 -export([parse/1, parse/2]).
 -export([address_count/2]).
 -export([contains/2]).
+-export([usort_cidrs/1]).
+-export([merge_cidrs/1]).
 -export([to_string/1]).
 -export([is_ipv4/1]).
 -export([is_ipv6/1]).
 
+-define(IPV4_MASK, 16#FFFFFFFF).
+-define(IPV6_MASK, 16#FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF).
+
 %% @doc parses S as a CIDR notation IP address and mask
 parse(S) ->
     parse(S, false).
 
 parse(S, Adjust) ->
-    {StartAddr, PrefixLen} = parse_cidr(S, Adjust),
+    {StartAddr, PrefixLen} = parse_cidr(S, Adjust),
```

Comment:
This appears to be a non-standard space that was breaking my editor. Not sure how that got into the code base.